doc/source/structures/vessels/vessel.rst

```diff
@@ -17,12 +17,6 @@ All vessels share a structure. To get a variable referring to any vessel you can
     // in case the target vessel changes.
     SET MY_VESS TO TARGET.
 
-.. note::
-
-    .. versionadded:: 0.13
-        A vessel is now a type of :struct:`Orbitable`. Much of what a Vessel can do can now by done by any orbitable object. The documentation for those abilities has been moved to the :ref:`orbitable page <orbitable>`.
-
-
```

Comment:
Partly Agree.  Move this out of a "versionadded" section, and change the phrasing so it doesn't say "is now", but don't totally delete the fact that this is mentioning that a vessel is a type of Orbitable, with the link present here too.

Reply:
Used a one-liner essentially the same as the one for celestial bodies, including the link.